Makefile

```diff
@@ -10,8 +10,9 @@ pre_commit_tests: ensure_pre_commit ## Run pre-commit tests
 
 .PHONY: test
 test: pre_commit_tests
-	go test -count 1 -v -timeout 90m ./test/...
+	bin/make-test
 
 .PHONY: clean
 clean:
 	rm -f .*.stamp
+	rm -f bin
```

Comment:
This got lost in the last PR. I'm not sure why we need to remove bin when running `make clean`. Also bin is a directory no a file so will likely error.

Reply:
👍 Will fix in PR #99